### DISCUSSION TITLE
`x-mask` - can't leave focues input

### DISCUSSION THREAD

**Discussion by adamkiss** [alpinejs/alpine#2853] [7. Bugs (archived)] [answered]
When you enter the masked field (or are autofocused into it), you can't leave the field.

Observed on: macos 10.15.7 + Safari 15.1
Doesn't happen on macos 10.15.7 + Evergreen Firefox DE

console doesn't include any errors or warnings.

https://user-images.githubusercontent.com/481362/164077250-98c34d80-5f26-4658-b519-e203e3862d20.mp4

**Comment by adamkiss** [comment, marked as answer]
Nevermind, I see Caleb is already working on it / fixed it.

**Comment by jottenlips** [comment]
Upgrading fixed the ability to click to a different focus. However, it is still initially focusing on our field on iOS.

**Comment by robert-stanciu** [comment]
Hi, any news on this issue?
It's still happening
